cmd/util/ledger/migrations/storage_v6.go

```diff
@@ -333,7 +335,7 @@ func (m *StorageFormatV6Migration) getDeferredKeys(payloads []ledger.Payload) ma
 						pathSeparator,
 					)
 
-					deferredOwner := dictionary.DeferredOwner().Bytes()
+					deferredOwner := dictionary.DeferredOwner()[:]
```

Comment:
so this was the error mostly for addresses which starts with some zeros?

Reply:
Yes, the only way I found this problem was because in your test case data, the owner had an address with leading zeros.

Removed the function in https://github.com/onflow/cadence/pull/1162